Tests/AppTests/Controllers/HeartbeatControllerTests.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  HeartbeatControllerTests.swift
+//  AppTests
+//
+//  Created by di on 30.11.2017.
+//
+
+import XCTest
+import Testing
+import HTTP
+import Sockets
+import Fluent
+@testable import Vapor
+@testable import App
+
+class HeartbeatControllerTests: TestCase {
+  
+  let drop = try! Droplet.testable()
+  let controller = HeartbeatController()
+  
+  func testThatPostSetBeatAnyValue() throws {
+    // arrange
+    let beat = 33
+    // act
+    guard let res = try setBeat(to: beat) as? Response else {
+      XCTFail()
+      return
+    }
+    // assert
+    try res.assertJSON("beat", equals: beat)
+  }
+  
+  func testThatRowCountAfterStoreAlwaysBeEqualOne() throws {
+    // arrange
+    let beat1 = 56
+    let beat2 = 33
+    let beat3 = 45
+    // act
+    _ = try setBeat(to: beat1)
+    let count1 = try Heartbeat.count()
+    //assert
+    XCTAssert(count1 == 1)
+    // act
+    _ = try setBeat(to: beat2)
+    let count2 = try Heartbeat.count()
+    // assert
+    XCTAssert(count2 == 1)
+    // act
+    _ = try setBeat(to: beat3)
+    let count3 = try Heartbeat.count()
+    // assert
+    XCTAssert(count3 == 1)
+  }
+  
+  func testThatShowGet204NoContentForEmptyBeatTable() throws {
+    // arange
+    try Heartbeat.makeQuery().delete()
+    let req = Request.makeTest(method: .get)
+    // act
+    let res = try controller.index(req: req).makeResponse()
+    // assert
+    res.assertStatus(is: .noContent)
+  }
+  
+  func testThatShowGetCurrentValueIfBeatTableIsNotEmpty() throws {
+    // arange
+    try Heartbeat.makeQuery().delete()
+    let beat = 45
+    _ = try setBeat(to: beat)
+    let req = Request.makeTest(method: .get)
+    // act
+    let res = try controller.index(req: req).makeResponse()
+    // assert
+    try res.assertJSON("beat", equals: beat)
+  }
+  
+  func testThatRoutePostMethodShouldSetAnyIntValue() throws {
+    // arrange
+    let beat = 488
```

Comment:
To not create `Int` values from your head every time, I suggest to add an extension for `Int` type. 
So you can use it like this `let beat = Int.randomValue`. and replace it in all tests

Reply:
fixed